### PR TITLE
Feature/store locator UI

### PR DESCRIPTION
**Pull request by Deepesh-emproto**
Screenshot from 2022-10-12 15-19-12

### CODE REVIEW

File: src/pages/StoreLocator/StoreLocator.component.tsx [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+   
+  return (
+     <div className={styles.store}>
+          <div className={styles.store_Main}>
+            <div className={styles.store_Main_Content}>
+              <div className={styles.store_Main_Content_Heading}> <b>{text.storeLocator.heading} </b></div>
+              <div className={styles.store_Main_Content_City}>
+                   <p className={styles.store_Main_Content_City_Text}>{text.storeLocator.subHeading}</p>
+                   <div>
+                        <Accordion accordionHeader={<AccordianHeader changed={handleExpand} Heading={heading}/>} accordionOptions={<AccordianOptions setHeading={setHeading}/>} expand={expand} className={styles.accordian_Class}/>
```
Comment (prateekoli1) [inline src/pages/StoreLocator/StoreLocator.component.tsx:43]: Use dropdown to select items not the accordion

File: src/pages/StoreLocator/StoreLocator.component.tsx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+     return (
+          <div onClick={changed} className={styles.accordianHeader}>
+              <p>{Heading}</p> 
+              <p><FaChevronDown/></p>
+          </div>
+     );
+ };
+   
+ 
+const AccordianOptions =(props:optionsValidate):JSX.Element=>{
```
Comment (prateekoli1) [inline src/pages/StoreLocator/StoreLocator.component.tsx:77]: Remove accordion, use dropdown to select city

File: src/components/StoreLocatorCard/StoreLocatorCard.component.tsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
```
Comment (prateekoli1) [inline src/components/StoreLocatorCard/StoreLocatorCard.component.tsx:1]: Rename component to StoreDetails

File: src/components/StoreDetail/StoreDetail.component.tsx [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+        <div>
+            <img src={picture} alt='text' height={'100%'} width={'100%'}/>
+        </div>
+        <div className={styles.card_breadth_text}>
+               <div className={styles.card_breadth_text_Heading}>{topText}</div>
+               <div className={styles.card_breadth_text_subHeading}>{address}</div>
+               <div className={styles.card_breadth_text_subText}>{text}</div>
+        </div>
+        <div className={styles.card_breadth_icon}>
+           <FaMapMarkerAlt className={styles.card_breadth_icon_height}/><p className={styles.card_breadth_icon_text}><b>Get Direction</b></p>
```
Comment (prateekoli9) [inline src/components/StoreDetail/StoreDetail.component.tsx:28]: onClick functionality is not implemented

File: src/components/DropDown/DropDown.component.tsx [hunk 5]
```diff
@@ -0,0 +1,20 @@
+type Props = {
```
Comment (prateekoli9) [inline src/components/DropDown/DropDown.component.tsx:1]: Remove thi scomponent

File: src/pages/StoreLocator/StoreLocator.component.tsx [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+   
+  return (
+     <div className={styles.store}>
+          <div className={styles.store_Main}>
+            <div className={styles.store_Main_Content}>
+              <div className={styles.store_Main_Content_Heading}> <b>{text.storeLocator.heading} </b></div>
+              <div className={styles.store_Main_Content_City}>
+                   <p className={styles.store_Main_Content_City_Text}>{text.storeLocator.subHeading}</p>
+                   <div>
+                        <DropDown dropdownTitle={<Title changed={handleExpand} Heading={heading}/>} dropDownOptions={<Options setHeading={setHeading} handleData={closeExpand}/>} expand={expand} className={styles.accordian_Class}/>
```
Comment (prateekoli9) [inline src/pages/StoreLocator/StoreLocator.component.tsx:47]: Use a dropdown not a dropdown component

**Pull request merged**